infrastructure/ansible/roles/ats/tasks/ats.yml

```diff
@@ -66,7 +66,7 @@
   environment:
     PATH: "{{ ansible_env.PATH }}:/bin:/usr/bin"
   shell:
-    executable: "{{ lookup('pipe','which bash') }}"
+    executable: "{{ lookup('pipe','/usr/bin/which bash') }}"
```

Comment:
I opted for using `type -p` instead of `which` in our build scripts since CentOS 7 and CentOS 8 do not include `which` by default.

Reply:
I'm fine with that. I thought `which` was part of LSB, but apparently not. But `type` is, so +1.

Not that the CentOS project cares much about adhering to the LSB standard, but IMO we should, and we should by default install CentOS packages necessary to fulfill it, and be ok using LSB things, IMO.

Saying "We adhere to and require the Linux Standard Base" is better than saying "we require whatever random stuff the CentOS devs decide to add or remove this version."

Reply:
Though, I don't fully understand the Ansible here (@jhg03a probably does), but I don't think `type` is what we need here, `type -t` just returns "alias." I think `which` here is needed to get the full path, not just whether it exists. It looks like `command -v` might be what we need, and it's part of POSIX.

Reply:
`command -v` sounds solid, too